### PR TITLE
LPS-195927 Update test suites names owned by platform experience

### PR DESCRIPTION
**Pull request by john-co**
**Description**
Include echo component transferred tests to platform experience. As part of this merge, update the mentions of peds with new team name platform experience

**Test Plan**
- [x] platform experience test suites run the appropriate tests

**JIRA Ticket**
https://liferay.atlassian.net/browse/LPS-195927

### CODE REVIEW

**Comment by john-co** [comment]
ci:test:configuration-frameworks

**Comment by john-co** [comment]
ci:test:platform-experience:nocompile

**Comment by john-co** [comment]
ci:test:platform-experience-accessibility

**Comment by john-co** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:platform-experience-acceptance:nocompile

**Comment by john-co** [comment]
pending QA review

includes: https://github.com/john-co/liferay-portal/pull/516#discussion_r1384019249

**Comment by john-co** [comment]
ci:stop

**Comment by john-co** [comment]
ci:test:configuration-frameworks:nocompile

**Comment by john-co** [comment]
ci:test:navigation-frameworks:nocompile

**Comment by john-co** [comment]
ci:test:platform-experience:nocompile

**Comment by john-co** [comment]
ci:test:configuration-framework:nocompile

**Comment by john-co** [comment]
ci:test:navigation-framework:nocompile

**Comment by john-co** [comment]
ci:test:platform-experience-accessibility:nocompile

**Comment by john-co** [comment]
ci:test:platform-experience-acceptance:nocompile

**Comment by john-co** [comment]
We may want to follow up with stabilizing some of these tests, especially look into configuration framework tests

File: ci.properties [hunk 1]
```diff
@@ -128,10 +130,9 @@ ci.test.available.suites=\
     osb-site-initializer-evp,\
     osb-site-initializer-partner-portal,\
     password-policies,\
-    peds,\
-    peds-acceptance,\
-    peds-accessibility,\
-    peds-sm,\
+    platform-experience,\
+    platform-experience-acceptance,\
+    platform-experience-accessibility,\
```
Comment (eltonccdantas) [inline ci.properties:135]: Much more intuitive.

**Review by eltonccdantas — APPROVED** [review]
LGTM! 😎

I would just rename the 1st or 2nd commit with a more intuitive name do differentiate them and squash the SF commit before forwarding.

**Comment by eltonccdantas** [comment]
> may want to follow up with stabilizing some of these tests, especially l

Alrighty.

**Comment by eltonccdantas** [comment]
Failures seem to be unrelated... Let's try to work on fixes for those tests. 🕵🏻 

`ci:test:configuration-frameworks`  [comparison run](https://testray.liferay.com/home/-/testray/runs/compare?formDate=1699364173645&testrayRunIdA=2452837033&testrayRunIdB=2453759926)

`ci:test:platform-experience-acceptance`   [comparison run](https://testray.liferay.com/home/-/testray/runs/compare?formDate=1699364656957&testrayRunIdA=2452837033&testrayRunIdB=2453758141)

`ci:test:platform-experience` [comparison run](https://testray.liferay.com/home/-/testray/runs/compare?formDate=1699364759455&testrayRunIdA=2452837033&testrayRunIdB=2453753644)

**Comment by john-co** [comment]
> I would just rename the 1st or 2nd commit with a more intuitive name do differentiate them and squash the SF commit before forwarding.

Thanks @eltonccdantas! Which commits specifically and what would you suggest the more intuitive name be?

**Comment by eltonccdantas** [comment]
> > I would just rename the 1st or 2nd commit with a more intuitive name do differentiate them and squash the SF commit before forwarding.
> 
> Thanks @eltonccdantas! Which commits specifically and what would you suggest the more intuitive name be?

I would add the file name to the commit message.
image

**Comment by john-co** [comment]
I see what you mean. I ended up just squashing the last two commits, I think that was my original intention.

**Comment by john-co** [comment]
ci:forward

**Pull request closed without merging**